pkg/cni/cni_test.go

```diff
@@ -391,9 +391,21 @@ func TestMain(m *testing.M) {
 		os.Exit(1)
 	}
 
+	testNetNsPath, err = ioutil.TempDir("", "virtcontainers-test-ns-")
```

Comment:
I think this won't work. The testNetNsPath will be overriden by testNetNsPath assignments in other test functions, won't it ?